packages/fileeditor-extension/schema/plugin.json

```
@@ -140,8 +142,44 @@
       }
     ]
   },
-  "title": "Text Editor",
-  "description": "Text editor settings.",
+  "jupyter.lab.toolbars": {
+    "HTML Viewer": []
```

```suggestion
    "Editor": []
```

I think based on:

https://github.com/jupyterlab/jupyterlab/blob/b782cd8f1ed775e93b4c78b865c50b819c97e0c2/packages/fileeditor-extension/src/commands.ts#L105

---

:+1: